.gitignore

```diff
@@ -139,7 +139,6 @@ cython_debug/
 
 # PyCharm IDE
 *.idea/
-.vscode/settings.json
```

Comment:
I think it would be better to leave this file in the .gitignore.
That way, a new clone will have these workspace settings, but personnal changes won't be accidentally commited to the repository.

Reply:
BTW, shouldn't we ignore the whole .vscode directory?

Reply:
Yes, I would ignore the whole `.vscode` directory!